po/de.po

```diff
@@ -25,7 +25,7 @@ msgstr ""
 
 #: app/flatpak-builtins-build-bundle.c:57
 msgid "Export runtime instead of app"
-msgstr "Laufzeitumgebung anstelle der Anwendung exportieren"
+msgstr "Laufzeit anstatt der Anwendung exportieren"
```

Comment:
I don't think this is a better translation

Reply:
I am confident it is. Not a massive difference anyways.

Reply:
You are exporting the Laufzeitumgebung.
You can not export a Laufzeit that's nonsense.

Reply:
May I kindly suggest to update it in the original string as well. It seems to be missing there.

"Export runtime environment instead of app"